jerry-core/ecma/builtin-objects/typedarray/ecma-builtin-typedarray-prototype.c

```diff
@@ -1172,6 +1218,12 @@ ecma_builtin_typedarray_prototype_sort (ecma_value_t this_arg, /**< this argumen
     goto free_values;
   }
 
+  if (ecma_arraybuffer_is_detached (info_p->array_buffer_p))
+  {
+    ecma_free_value (sort_value);
```

Comment:
There is an assert below that `sort_value == ECMA_VALUE_EMPTY` so don't need to free it. Please move the isDetached check after this assert.

Reply:
This is still not fixed.